intranet/apps/eighth/management/commands/dev_create_blocks.py

```diff
@@ -0,0 +1,59 @@
+from datetime import date, timedelta
+
+from django.core.management.base import BaseCommand, CommandError
+
+from intranet.apps.eighth.models import EighthActivity, EighthBlock, EighthScheduledActivity
+
+
+class Command(BaseCommand):
+    help = "Generate 8th Period Blocks"
+
+    def add_arguments(self, parser):
+        # Positional arguments
+        parser.add_argument("end_date")
+
+    def handle(self, *args, **kwargs):
+        try:
+            end_month, end_day, end_year = kwargs["end_date"].split("/")
```

Comment:
This works if the provided date is in the correct format. However, what if the user passes `13/15/2020`? That case would pass this `try`-`except`, yet fail later. I would recommend setting `unaware_day` to `datetime.datetime.strptime(kwargs["end_date"], "%m/%d/%Y")` and catch any `ValueError` ([Reference](https://docs.python.org/3/library/datetime.html?highlight=datetime#datetime.datetime.strptime)).
You can then convert this naive `datetime` object to an aware object with `django.utils.timezone.make_aware`.